.github/CODEOWNERS

```diff
@@ -30,813 +30,810 @@
 
-# ServiceLabel: %DevOps %Service Attention
-#/<NotInRepo>/          @narula0781 @ashishonce @romil07
+# ServiceLabel: %Managed Identity
+# ServiceOwners:                                     @varunkch
 
-# ServiceLabel: %Dev Spaces %Service Attention
-#/<NotInRepo>/          @yuzorMa @johnsta @greenie-msft
+# ServiceLabel: %Machine Learning
+# ServiceOwners:                                     @azureml-github
 
-# ServiceLabel: %Devtestlab %Service Attention
-#/<NotInRepo>/          @Tanmayeekamath
+# ServiceLabel: %Machine Learning Compute
+# ServiceOwners:                                     @azureml-github
 
-# ServiceLabel: %Device Provisioning Service %Service Attention
-#/<NotInRepo>/          @nberdy
+# ServiceLabel: %Machine Learning Experimentation
+# ServiceOwners:                                     @aashishb
 
-# ServiceLabel: %Digital Twins %Service Attention
-#/<NotInRepo>/          @@drwill-ms @@timtay-microsoft @@abhipsaMisra @@vinagesh @@azabbasi @@barustum @@jamdavi
+# ServiceLabel: %Managed Services
+# ServiceOwners:                                     @Lighthouse-Azure
 
-# ServiceLabel: %Functions %Service Attention
-#/<NotInRepo>/          @ahmedelnably @fabiocav
+# ServiceLabel: %MariaDB
+# ServiceOwners:                                     @ambhatna @savjani
 
-# ServiceLabel: %Graph.Microsoft %Service Attention
-#/<NotInRepo>/          @dkershaw10 @baywet
+# ServiceLabel: %Marketplace Ordering
+# ServiceOwners:                                     @prbansa
 
-# ServiceLabel: %Guest Configuration %Service Attention
-#/<NotInRepo>/          @mgreenegit @vivlingaiah
+# ServiceLabel: %Media Services
+# ServiceOwners:                                     @akucer
 
-# ServiceLabel: %HDInsight %Service Attention
-#/<NotInRepo>/          @aim-for-better @idear1203 @deshriva
+# ServiceLabel: %Migrate
+# ServiceOwners:                                     @shijojoy
 
-# ServiceLabel: %HPC Cache %Service Attention
-#/<NotInRepo>/          @romahamu @omzevall
+# PRLabel: %Mixed Reality Authentication
+/sdk/mixedreality/azure-mixedreality-authentication/ @RamonArguelles
 
-# ServiceLabel: %Import Export %Service Attention
-#/<NotInRepo>/          @madhurinms
+# ServiceLabel: %Mobile Engagement
+# ServiceOwners:                                     @kpiteira
 
-# ServiceLabel: %KeyVault %Service Attention
-#/<NotInRepo>/          @RandalliLama @schaabs @jlichwa
+# PRLabel: %Models Repository
+/sdk/modelsrepository/                               @drwill-ms @timtay-microsoft @abhipsaMisra @digimaun @andyk-ms @brycewang-microsoft @tmahmood-microsoft @ngastelum-ms
 
-# ServiceLabel: %Kubernetes Configuration %Service Attention
-#/<NotInRepo>/          @NarayanThiru
+# PRLabel:  %Monitor
+/sdk/monitor/azure-monitor-query*/                   @srnagar @jairmyree @lmolkova @Azure/azure-sdk-write-monitor-data-plane
+/sdk/monitor/azure-monitor-ingestion*/               @srnagar @jairmyree @lmolkova @Azure/azure-sdk-write-monitor-data-plane
```

Comment:
This will not have a PR label associated.  The comment applies only to the path that immediately follows.